Representor/Transition.swift

```diff
@@ -30,27 +30,14 @@ public func ==<T : AnyObject>(lhs:InputProperty<T>, rhs:InputProperty<T>) -> Boo
 public typealias InputProperties = Dictionary<String, InputProperty<AnyObject>>
 
 /** Transition instances encapsulate information about interacting with links and forms. */
-public struct Transition : Equatable, Hashable {
-  public let uri:String
+public protocol Transition : Equatable, Hashable {
```

Comment:
Thanks for picking up on that, I certainly did this the wrong way round `<TransitionType : Transition>`.